### DISCUSSION TITLE
Export to Thread

### DISCUSSION THREAD

**Discussion by Toriandra** [Tyrrrz/DiscordChatExporter#742] [General]
Is this something that could one day be used to move a portion of a channel into a thread?

**Comment by 96-LB** [comment]
Discord does not support importing channels, so there is unfortunately nothing DCE can do about this. Your best bet would be to "recreate" it by resending messages. It could theoretically be possible to write a bot which reads the JSON export and sends identical messages, but as far as I know no such tool exists right now.
